app/services/rights_labeler.rb

```diff
@@ -2,48 +2,35 @@
 
 # Parses rights XML to produce a label
 class RightsLabeler
-  # @param [String] rights_xml rights metadata serialized as XML in string form
+  # @param [Cocina::Models::AdminPolicyDefaultAccess] default_access admin policy default access instance
   # @return [String] a label expressing the rights of the object
-  def self.label(rights_xml)
-    new(rights_xml).label
+  def self.label(default_access)
+    new(default_access).label
   end
 
-  # @param [String] rights_xml rights metadata serialized as XML in string form
-  def initialize(rights_xml)
-    @rights_xml = Nokogiri::XML(rights_xml)
+  # @param [Cocina::Models::AdminPolicyDefaultAccess] default_access admin policy default access instance
+  def initialize(default_access)
+    @default_access = default_access
   end
 
   # @return [String] a label expressing the rights of the object
   def label
-    machine_read_access_node = rights_xml.search('//rightsMetadata/access[@type="read"]/machine').first
-    machine_discover_access_node = rights_xml.search('//rightsMetadata/access[@type="discover"]/machine').first
+    return '' if default_access.nil? || default_access.access.nil?
```

Comment:
We have at least one test that breaks if we don't have a case for returning an empty label, and I don't want to change this behavior in this PR.